src/cartesian-chart/__tests__/cartesian-chart-error-bars.test.tsx

```diff
@@ -31,7 +31,7 @@ describe("CartesianChart: errorbar series", () => {
       expect(getAllTooltipSeries()).toHaveLength(1);
       expect(getTooltipSeries(0).findKey().getElement().textContent).toBe("Column 1");
       expect(getTooltipSeries(0).findValue().getElement().textContent).toBe("2");
-      expect(getTooltipSeries(0).findDescription().getElement().textContent).toBe("Error range1 - 3");
+      expect(getTooltipSeries(0).findDescription()!.getElement().textContent).toBe("Error range1 - 3");
```

Comment:
Before, the `!` were not necessary because the description could never be empty, but it should be able to.